libkbfs/folder_branch_ops.go

```diff
@@ -5642,9 +5642,19 @@ func (fbo *folderBranchOps) finalizeResolutionLocked(ctx context.Context,
 		fbo.fbm.archiveUnrefBlocks(irmd.ReadOnly())
 	}
 
+	mdCopyWithLocalOps, err := md.deepCopy(fbo.config.Codec())
+	if err != nil {
+		return err
+	}
+	mdCopyWithLocalOps.data.Changes.Ops = newOps
+	irmdCopyWithLocalOps := MakeImmutableRootMetadata(
+		mdCopyWithLocalOps, session.VerifyingKey, mdID,
```

Comment:
the `mdID` doesn't match up with the copied MD, right? In that case, I think it's better to make notifyOneOpLocked take a `ReadOnlyRootMetadata`, since I don't think it actually depends on the mdID.